easybuild/easyconfigs/t/TEtranscripts/TEtranscripts-2.2.0-foss-2020a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'TEtranscripts'
+version = '2.2.0'
+
+homepage = 'https://github.com/mhammell-laboratory/TEtranscripts'
+description = """TEtranscripts and TEcount takes RNA-seq (and similar data) and annotates reads
+to both genes & transposable elements.
+TEtranscripts then performs differential analysis using DESeq2."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://github.com/mhammell-laboratory/TEtranscripts/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['9060f284ca20d598cfb6cc6895d5708afdd7850b353359fbe32e893f29bda98e']
+
+multi_deps = {'Python': ['3.8.2', '2.7.18']}
+
+dependencies = [
+    ('Pysam', '0.16.0.1'),
+    ('R-bundle-Bioconductor', '3.11', '-R-4.0.0'),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+options = {'modulename': 'TEToolkit'}
+
+fix_python_shebang_for = ['bin/*']
+
+sanity_check_paths = {
+    'files': ['bin/TEtranscripts', 'bin/TEcount'],
```

Comment:
@hajgato When binaries/scripts are involved, it's a good idea to include a sanity check command if possible (something like `TEcount --version` or `TEcount --help`) via `sanity_check_commands`, especially if `multi_deps` is involved.

Reply:
fixed in #10921